recipes-aos/aos-communicationmanager/aos-communicationmanager_git.bb

```diff
@@ -69,6 +69,9 @@ python do_update_config() {
     data["IAMProtectedServerURL"]= node_hostname+":8089"
     data["IAMPublicServerURL"] = node_hostname+":8090"
 
+    # Update DNS IP
+    data["DNSIp"] = d.getVar("AOS_DNS_IP")
```

Comment:
DNSIP

Probably  AOS_DNS_IP should have some default value? Or if not defined, do not set this option at all?